### PR TITLE
Remove absolute file paths from theme-config using the deploy_path

### PR DESCRIPTION
**Pull request by peterjaap**
Bug fix for https://github.com/deployphp/deployer/issues/2754

### CODE REVIEW

File: recipe/shopware.php [hunk 1]
```diff
@@ -159,8 +159,17 @@ function getPlugins(): array
...
-        runLocally('sed -i "" -E \'s/\\\\\/var\\\\\/www\\\\\/htdocs\\\\\/releases\\\\\/[0-9]+\\\\\///g\' files/theme-config/* || true');
+        // See:
+        // - https://github.com/shopware/platform/commit/01c8ff86c7d8d3bee1888a26c24c9dc9b4529cbc
+        // - https://issues.shopware.com/issues/NEXT-17720
+        // - https://github.com/deployphp/deployer/issues/2754
+        $deployPath = get('deploy_path');
+        if (substr($deployPath, -1, 1) !== '/') {
+            $deployPath .= '/';
+        }
+        $deployPath .= 'releases/[0-9]*/';
```
Comment (antonmedv) [inline recipe/shopware.php:170]: Let’s use `releases/.+?/`.

Comment (peterjaap) [reply recipe/shopware.php]: That doesn't work, it strips off the whole path;

```
sed -E 's#\\/var\\/www\\/htdocs\\/releases\\/.+?\\/##g' test.json

[
    {
        "storefront": {
            "path": "Resources\/app\/storefront\/src",
            "entryFilePath": "Resources\/app\/storefront\/src\/main.js",
            "webpack": null,
            "styleFiles": [
                "base.scss",
                "_base.scss",
                "@Plugins"
            ]
        }
    }
]%  
```

Comment (peterjaap) [reply recipe/shopware.php]: @antonmedv OK I changed the regex to;

`$deployPath .= 'releases/[0-9a-zA-Z]*/';`

Should be good to merge now :+1:

**Comment by antonmedv** [comment]
Merge?

**Pull request merged**